pandas/tests/groupby/test_allowlist.py

```diff
@@ -100,131 +95,6 @@ def test_regression_allowlist_methods(raw_frame, op, axis, skipna, sort):
         tm.assert_frame_equal(result, expected)
 
 
-def test_groupby_blocklist(df_letters):
```

Comment:
Removing this test - there we no longer try to get to ops through `__getattr__`.